samples/matter/lock/boards/nrf7002dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&led0 {
```

Comment:
Can we contribute it to generic board configuration, like e.g. for nRF52840?
https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts#L26

Reply:
Yeah I found on nRF7002dk schematic that the .dts configuration is wrong. I pushed separate PR fixing it to the sdk-nrf.